test/integration/test-custom-filename.js

```diff
@@ -8,12 +8,14 @@ var assert       = common.assert;
 var mime         = require('mime-types');
 var http         = require('http');
 var fs           = require('fs');
+var path         = require('path');
 
 var FormData     = require(common.dir.lib + '/form_data');
 var IncomingForm = require('formidable').IncomingForm;
 
-var knownFile = common.dir.fixture + '/unicycle.jpg';
-var unknownFile = common.dir.fixture + '/unknown_file_type';
+var knownFile = path.join(common.dir.fixture, 'unicycle.jpg');
+var unknownFile = path.join(common.dir.fixture, 'unknown_file_type');
+var relativeFile = path.relative(path.join(__dirname, '..'), knownFile);
```

Comment:
Changed this to be a relative path to `knownFile`. So it will have the OS-specific separator. This makes the test more meaningful than a hardcoded string. LMK if you prefer a change.